invenio_migrator/legacy/records.py

```diff
@@ -95,12 +110,17 @@ def get(query, from_date, **kwargs):
     return len(recids), recids
 
 
-def dump(recid, from_date, with_json=True, latest_only=False, **kwargs):
+def dump(recid, from_date, with_json=True, latest_only=False,
```

Comment:
Since it's not possible to turn off ``with_json`` with a CLI arg, I would change it to ``skip_json=False``